variables.tf

```diff
@@ -1,3 +1,9 @@
+variable "alarm_suffix" {
+  type        = string
+  default     = ""
+  description = "Suffix to add to alarm name, used for separating different AWS account."
+}
+
```

Comment:
Yesss...alphabetize it... *rubs hands together*

Reply:
I....I.....It's _beautiful_!